src/component/AddItem.js

```diff
@@ -81,7 +88,13 @@ const AddItem = ({ firestore }) => {
   const handleSubmit = event => {
     event.preventDefault();
     let normalizedName = normalizeName(name);
-    addItem(normalizedName, numberOfDays);
+    // This conditional allows the add button to be accidentally clicked
+    // while the form is empty without throwing an error
+    if (normalizedName === '') {
```

Comment:
Hmm. This code should have been in `master` already when you created this branch, right? This is a change from last week, if I recall correctly. I'm a bit confused about the state of this branch 😕

Reply:
See my response above 😕